src/main/scala/io/iohk/ethereum/vm/Memory.scala

```diff
@@ -70,6 +70,21 @@ class Memory private(private val underlying: ByteString) {
       (newUnderlying.slice(start, end), new Memory(newUnderlying))
     }
 
+  /** This function will expand the Memory size as if storing data given the `offset` and `size`.
+    * If the memory is already initialised at that region it will not be modified, otherwise it will be filled with
+    * zeroes.
+    * This is required to satisfy memory expansion semantics for *CALL* opcodes.
+    */
+  def expand(offset: UInt256, size: UInt256): Memory = {
+    val totalSize = (offset + size).toInt
+    if (this.size >= totalSize || size.isZero)
```

Comment:
BTW, we were talking about the same equation 😄  
Referencing the YP version like `YP EIP-150 REVISION` is not enough cause I also have this revision, and my eq numbers are different. My version is `EIP-150 REVISION (1e18248 - 2017-04-12)`